### PR TITLE
Kill all traces for esr31 builds (#614)

### PR DESCRIPTION
**Pull request by whimboo**
This PR will fix issue #614. @davehunt maybe you can also review? Thanks.

### CODE REVIEW

File: config/ondemand/update-override-update-url.ini [hunk 1]
```diff
@@ -11,4 +11,4 @@ override-update-url=https://aus4.mozilla.org/update/3/%PRODUCT%/%VERSION%/%BUILD
 
 [linux ubuntu 14.04 64bit]
 platform=linux64
-31.0b8=en-US
+38.0=en-US
```
Comment (davehunt) [inline config/ondemand/update-override-update-url.ini]: Should this be esr?

**Comment by davehunt** [comment]
r+ lgtm

File: config/ondemand/update-override-update-url.ini (hunk 1 above)
Comment (whimboo) [inline config/ondemand/update-override-update-url.ini]: We might wanna do that, yes. I will update before squashing.

**Comment by whimboo** [comment]
Merged as https://github.com/mozilla/mozmill-ci/commit/04e04681f6ea6aff6f1e7fd2d217b871abf041c9

**Pull request closed without merging**